docs/using-the-jdbc-driver/using-plugins/UsingTheReaderClusterConnectionPlugin.md

```diff
@@ -0,0 +1,10 @@
+# Reader Cluster Connection Plugin
+
+When making connections to the reader cluster endpoint of an Aurora database, each connection is made to a random available reader instance. This may cause errors if operations depend on making connections to the same instance. For example, to execute a query timeout with the MySQL Connector/J 8.0 Driver, a new internal connection will be made to kill the initial query. If the reader endpoint is used, the kill query may fail to cancel the initial query because the queries will be made to different hosts. The Reader Cluster Connection Plugin will replace the cluster endpoint connection with an instance connection.
```

Comment:
We should perhaps add a paragraph to explain the behavior once the plugin is enabled. Is any reader picked initially? Are all the connections after going to that same reader? What happens if that plugin and read write splitting are enabled?